zanata-war/src/main/java/org/zanata/action/MergeTransAction.java

```diff
@@ -184,6 +186,9 @@ public void startMergeTranslations() {
         mergeTranslationsManager.start(sourceProjectSlug,
             sourceVersionSlug, targetProjectSlug, targetVersionSlug,
             !keepExistingTranslation);
+        FacesContext context = FacesContext.getCurrentInstance();
+        NavigationHandler navigationHandler = context.getApplication().getNavigationHandler();
+        navigationHandler.handleNavigation(context, null, "merge-translation");
```

Comment:
This is in an AJAX context not a normal form action. I think this is the only place that requires this at the moment.